### PR TITLE
Home task: HTML 01

### PR DESCRIPTION
**Pull request by sergey-mario**

### CODE REVIEW

File: smaretski/01_HTML/task_01.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Пример страницы</TITLE> 
+    </HEAD>
+<!-- это начало документа-->  
+    <BODY> 
+        <H1>Это заголовок</H1> <!--это заголовок страницы-->
+        <P><DEL>Добрый день, <EM>Уважаемые слушатели</DEL></EM>! Сегодня мы с вами будем рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P> 
+        <P>Далее мы будем использовать произвольный текст для наполнения контентом. Не стоит вдумываться в его содержание.</P>
+        <P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения, длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года. <!--Демонстрация полноценных серий мультсериала началась 17 декабря 1989 года на канале FOX. Выпускается «Gracie Films» для кинокомпании «20th Century Fox». Мультсериал в настоящее время известен телезрителям более чем 100 стран.</P>
```
Comment (pluhin) [inline smaretski/01_HTML/task_01.html]: You didn't close tag p here

File: smaretski/01_HTML/task_04.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <pre>print len(months)</pre>
+        <p>которая вернёт 12.</p>
+        <p>Ещё один пример списков — категории в поваренной книге. Например…</p>
+        <pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+        <p>Таким образом, в <b>categories[0]</b> хранится <i>'Main dish'</i>, а в <b>categories[4]</b> — <i>'Cookies'</i>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+        <h2>Комментарии</h2>
+        <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+        <p>&#35</p>
```
Comment (pluhin) [inline smaretski/01_HTML/task_04.html]: it isn't paragraph, it is code section

File: smaretski/01_HTML/task_05.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            <hr>
+            <h2>Функции</h2>
+            <p>Когда мы импортировали модуль random, мы использовали функцию <b>randint()</b>. Функция — это блок кода, который создан для того, чтобы вызывать его в программе, обычно, более одного раза. Это упрощает работу над программой и предотвращает набор одного и того же кода снова и снова. Грубо говоря, если вам нужно написать некоторый код больше, чем 1-2 раза, стоит сделать его функцией. Хотя следующие два примера очень простые, это хорошие примеры функций. Теперь мы хотим, скажем, взять два числа, сложить их, перемножить, вычесть одно из другого, затем вывести на экран эти числа и результаты одновременно. Допустим, нам нужно сделать это три раза для трёх наборов чисел. Тогда наш пример будет выглядеть как код ниже.</p>
+            <pre>#silly example
+print '%d + %d = %d ' % (1,2,1+2)
+print '%d * %d = %d ' % (1,2,1*2)
+print '%d - %d = %d ' % (1,2,1-2)
+print '\n'
+<p>print '%d + %d = %d ' % (1,4,1+4)
```
Comment (pluhin) [inline smaretski/01_HTML/task_05.html]: what is paragraph doing here? Inside code section

**Review by pluhin — CHANGES_REQUESTED** [review]

File: smaretski/01_HTML/task_01.html (hunk 1 above)
Comment (sergey-mario) [reply smaretski/01_HTML/task_01.html]: fixed

File: smaretski/01_HTML/task_04.html (hunk 2 above)
Comment (sergey-mario) [reply smaretski/01_HTML/task_04.html]: fixed

File: smaretski/01_HTML/task_05.html (hunk 3 above)
Comment (sergey-mario) [reply smaretski/01_HTML/task_05.html]: fixed and used tag <pre>

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**